src/bootstrap-decorator.js

```diff
@@ -52,4 +52,12 @@ function(decoratorsProvider, sfBuilderProvider, sfPathProvider) {
     'default': {template: base + 'default.html', builder: defaults}
   }, []);
 
-}]);
+}])
+    .controller('TextAreaController', ['$scope', 'marked', function ($scope, marked) {
+      $scope.render = function (text) {
+        if (text) {
+          return marked(text);
```

Comment:
It comes from an ng-obiba dependency : 'angular-marked' which is already a dependency in mica2